Makefile

```diff
@@ -62,7 +65,7 @@ clean-build:
 	@rm -rf build | true
 clean-java:
 	@echo " - cleaning java resources"
-	@cd src && mvn clean > /dev/null && cd ..
+	@cd src && mvn --quiet clean > /dev/null && cd ..
```

Comment:
--quite flag is not working as expected. May be we can follow up in the next improvement

Reply:
do we need to remove it or it sitill works with it ?